### DISCUSSION TITLE
Future of Redis Usage

### DISCUSSION THREAD

**Discussion by jrtitus** [edgexfoundry/community#268] [Q&A]
With the recent change of Redis from [BSD](https://github.com/redis/redis/blob/d2c8a4b91e8c0e6aefd1f5bc0bf582cddbe046b7/COPYING) to [RSALv2/SSPLv1](https://github.com/redis/redis/blob/804110a487f048669aa9d9412e5789ec43f4fe39/LICENSE.txt) are there talks within EdgeX Foundry to migrate to one of the Redis forks such as [Valkey](https://github.com/valkey-io/valkey) or [Redict](https://codeberg.org/redict/redict)? (I presume Valkey may be the path of least resistance.) Or will Redis 7, prior to the license change, continue to be used for the foreseeable future?

**Comment by cloudxxx8** [comment]
yes, we plan to replace Redis with other alternative database, and we are still discussing it and try to make a comparison table.
